### PR TITLE
Release 30 validation

### PR DESCRIPTION
**Pull request by renanrodrigo**
Test adjustments to the release-30 validation.
No changes to functionality, only on test expectations.

### CODE REVIEW

**Comment by renanrodrigo** [comment]
The mypy issue is new, and should not block those changes.
It will be addressed against the `main` branch in a separate PR

**Comment by renanrodrigo** [comment]
@orndorffgrant branch updated

**Review by orndorffgrant — APPROVED** [review]
New commits LGTM

File: features/landscape.feature [hunk 1]
```diff
@@ -64,27 +64,35 @@ Feature: Enable landscape on Ubuntu
...
+        # TODO: Known "Uninstalling" bug: #2839
         When I run `sudo pro disable landscape` with sudo
         Then I will see the following on stdout:
         """
         Executing `landscape-config --disable`
         Failed running command 'landscape-config --disable' [exit(1)]. Message: error: config file /etc/landscape/client.conf can't be read
         Backing up /etc/landscape/client.conf as /etc/landscape/client.conf.pro-disable-backup
         [Errno 2] No such file or directory: '/etc/landscape/client.conf' -> '/etc/landscape/client.conf.pro-disable-backup'
-        Uninstalling landscape-client
+        Uninstalling
```
Comment (renanrodrigo) [inline features/landscape.feature:75]: lol

Comment (orndorffgrant) [reply features/landscape.feature:75]: Yeah... #2839

**Review by orndorffgrant — APPROVED** [review]
Last commit LGTM

**Comment by renanrodrigo** [comment]
wow this PR is so approved I am happy now

**Pull request merged**